builtin/game/statbars.lua

```diff
@@ -9,8 +9,8 @@ local health_bar_definition = {
 	number = core.PLAYER_MAX_HP_DEFAULT,
 	item = core.PLAYER_MAX_HP_DEFAULT,
 	direction = 0,
-	size = {x = 24, y = 24},
-	offset = {x = (-10 * 24) - 25, y = -(48 + 24 + 16)},
+	size = {x = 22, y = 22},
```

Comment:
This is probably problematic due to #2419 plus game/texture authors assuming that 24x hearts will render pixel perfect.